src/plugins/home/public/application/components/guided_onboarding/use_case_card.tsx

```diff
@@ -46,7 +46,7 @@ const constants: UseCaseConstants = {
       }),
       description: i18n.translate('home.guidedOnboarding.gettingStarted.search.cardDescription', {
         defaultMessage:
-          'Create a search experience for your websites, applications, workplace content, or anything in between.',
+          'Create a search experience for your websites, applications, workplace content, and more.',
```

Comment:
```suggestion
          'Create a finely-tuned search experience for your websites, applications, workplace content, and more.',
```

Looking to expand this, so that it's closer in length to the other two descriptions. 
```suggestion
          'Create a search experience for your websites, applications, workplace content, and more.',
```

Reply:
@kellyemurphy here's a new screenshot with the change:

<img width="1087" alt="Screen Shot 2022-06-27 at 8 42 39 PM" src="https://user-images.githubusercontent.com/5226211/176062926-2bd81c5a-ba40-441b-ae93-954fc3cb6dbd.png">

Reply:
Drive by comment, but should we be using solution icons for these use cases?

Reply:
> Drive by comment, but should we be using solution icons for these use cases?

@johnbarrierwilson can you comment on this?

Reply:
I see John is OOO this week. I will follow-up with him on this and open up a separate PR if any changes are needed.

Reply:
Just to close the loop on this - there are updated icons available. I will open a separate PR to fix.